### PR TITLE
chore: prepend cli-core for util-mock and awscloudformation

### PR DESCRIPTION
**Pull request by evcodes**
#### Description of changes

Replace all imports in util-mock and awscloudformation packages, (from 'amplify-cli-core' -> from '@aws-amplify/amplify-cli-core')
#### Checklist

- [x] PR description included
- [x] `yarn test` passes
- [x] Tests are [changed or added](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#tests)
- [ ] Relevant documentation is changed or added (and PR referenced)
- [ ] New AWS SDK calls or CloudFormation actions have been added to relevant test and service IAM policies
- [ ] [Pull request labels](https://github.com/aws-amplify/amplify-cli/blob/dev/CONTRIBUTING.md#labels) are added

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: packages/amplify-provider-awscloudformation/src/__tests__/permission-boundary/permissions-boundary.test.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ const permissionsBoundaryArn = 'arn:aws:iam::123456789012:policy/some-policy-nam
 const argName = 'permissions-boundary';
 const envName = 'newEnvName';
 
-// jest.mock('amplify-cli-core');
+// jest.mock('@aws-amplify/amplify-cli-core');
```
Comment (sobolk) [inline packages/amplify-provider-awscloudformation/src/__tests__/permission-boundary/permissions-boundary.test.ts:11]:
```suggestion
```

File: packages/amplify-provider-awscloudformation/src/amplify-service-manager.ts [hunk 2]
```diff
@@ -2,12 +2,12 @@ import * as fs from 'fs-extra';
 import * as path from 'path';
 import inquirer from 'inquirer';
 import sequential from 'promise-sequential';
-import { $TSContext } from 'amplify-cli-core';
+import { $TSContext } from '@aws-amplify/amplify-cli-core';
```
Comment (sobolk) [inline packages/amplify-provider-awscloudformation/src/amplify-service-manager.ts:5]:
```suggestion
```

File: packages/amplify-provider-awscloudformation/src/amplify-service-manager.ts [hunk 3]
```diff
@@ -2,12 +2,12 @@ import * as fs from 'fs-extra';
...
 import { S3 } from './aws-utils/aws-s3';
 import { getConfiguredAmplifyClient } from './aws-utils/aws-amplify';
 import { ProviderName, AmplifyAppIdLabel } from './constants';
 import { checkAmplifyServiceIAMPermission } from './amplify-service-permission-check';
-import { stateManager, AmplifyFault, AmplifyError } from 'amplify-cli-core';
+import { stateManager, AmplifyFault, AmplifyError } from '@aws-amplify/amplify-cli-core';
```
Comment (sobolk) [inline packages/amplify-provider-awscloudformation/src/amplify-service-manager.ts:10]:
```suggestion
import { $TSContext, stateManager, AmplifyFault, AmplifyError } from '@aws-amplify/amplify-cli-core';
```

**Review by sobolk — DISMISSED** [review]

**Review by sobolk — APPROVED** [review]

**Review by sobolk — DISMISSED** [review]

**Comment by evcodes** [comment]
e2e test run:

File: packages/amplify-util-mock/src/__tests__/func/index.test.ts [hunk 4]
```diff
@@ -1,7 +1,7 @@
-import { $TSAny } from 'amplify-cli-core';
+import { $TSAny } from '@aws-amplify/amplify-cli-core';
 import { start } from '../../func';
 import { getInvoker, getBuilder } from '@aws-amplify/amplify-category-function';
-import { stateManager } from 'amplify-cli-core';
+import { stateManager } from '@aws-amplify/amplify-cli-core';
```
Comment (edwardfoyle) [inline packages/amplify-util-mock/src/__tests__/func/index.test.ts:4]: combine?

**Review by edwardfoyle — APPROVED** [review]

**Review by sobolk — APPROVED** [review]

**Pull request merged**